pkg/handler/handler.go

```diff
@@ -19,11 +19,14 @@ import (
 	"io"
 	"net/http"
 
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
 	"github.com/kanisterio/kanister/pkg/version"
 )
 
 const (
 	healthCheckPath = "/v0/healthz"
+	metricsPath     = "/v0/metrics"
```

Comment:
Probably we should use `/metrics` path instead so that it'll be in line with other k10 services

Reply:
As discussed, changing the path to `/metrics`